docs/content/developer/getting-started/simple-token-transfer.mdx

```diff
@@ -0,0 +1,438 @@
+- Uses the `coin::create_currency` function to create the token by specifying the name, symbol, and decimal precision of the token.
+- The `coin::create_currency` function returns a TreasuryCap, which allows the owner to mint and transfer the token, and the metadata of the token.
+- Transfers the metadata to the owner of the contract using `transfer::public_freeze_object(metadata)` and `transfer::public_transfer(treasury, ctx.sender())`.
+
+```move reference
+https://github.com/iota-community/token-transfer-tutorial/blob/ea71112c1156d1885567353fee0a03e09308e293/sources/token_transfer.move#L9-L13
+```
+
+#### [`mint`](https://github.com/iota-community/token-transfer-tutorial/blob/ea71112c1156d1885567353fee0a03e09308e293/sources/token_transfer.move#L15-L23)
+
+- Mints the token and transfers it to the recipient address.
+- Can only be called by the owner of the contract.
+- Creates a new object of the `MY_TOKEN` token using the TreasuryCap.
+- The TreasuryCap address (which was generated during the contract's publication) is required to mint the tokens.
+- Transfers the minted tokens to the recipient after they are created.
+
+```move reference
+https://github.com/iota-community/token-transfer-tutorial/blob/ea71112c1156d1885567353fee0a03e09308e293/sources/token_transfer.move#L15-L23
+```
+
+#### [`transfer`](https://github.com/iota-community/token-transfer-tutorial/blob/ea71112c1156d1885567353fee0a03e09308e293/sources/token_transfer.move#L25-L33)
+
+- Transfers the token from the caller address to the recipient address.
+- Requires the correct token object address that is owned by the caller.
+- The function first uses the `coin::split` function to split the specified amount from the token and then uses `transfer::public_transfer` to transfer the split tokens to the recipient.
+
+```move reference
+https://github.com/iota-community/token-transfer-tutorial/blob/ea71112c1156d1885567353fee0a03e09308e293/sources/token_transfer.move#L25-L33
+```
+
+#### [`balance`](https://github.com/iota-community/token-transfer-tutorial/blob/ea71112c1156d1885567353fee0a03e09308e293/sources/token_transfer.move#L34-L37)
+
+- Returns the balance of the token for the specified owner.
+- Requires passing the token address to retrieve the balance.WW
+- Uses `coin::balance` to query the balance and returns the value held by the coin object.
+
+```move reference
+https://github.com/iota-community/token-transfer-tutorial/blob/ea71112c1156d1885567353fee0a03e09308e293/sources/token_transfer.move#L34-L37
+```
+
+Navigate to the [`source/token-transfer.move`](https://github.com/iota-community/token-transfer-tutorial/blob/ea71112c1156d1885567353fee0a03e09308e293/sources/token_transfer.move#L1-L38) file and paste the following code:
+
+```move reference
+https://github.com/iota-community/token-transfer-tutorial/blob/ea71112c1156d1885567353fee0a03e09308e293/sources/token_transfer.move#L1-L38
+``` 
+
+### Build the Package
+
+Once the contract is ready, build the package by running:
```

Comment:
Link to https://docs.iota.org/developer/getting-started/build-test

Reply:
Updating it.